packages/web/docs/src/app/federation/contact-an-expert-button.tsx

```diff
@@ -0,0 +1,17 @@
+'use client';
+
+import { CallToAction } from '@theguild/components';
+
+export function ContactAnExpertButton() {
```

Comment:
It's used in one place, why do we need a dedicated file for a button?

Reply:
we can't pass functions from server components, so this button was extracted and marked as client component.

> [!note]
>
> we can't mark all page as client because we couldn't use Next.js metadata API `export const metadata = { title ... }`

Reply:
@dimaMachina You can import `ContactButton` from `@theguild/components`. it accepts children and also works as an anchor as a fallback when crisp is not mounted (previews, users without JavaScript, adblockers).